### PR TITLE
 dev/reference/tools: update MySQL settings in Syncer

### PR DESCRIPTION
**Pull request by huangxiuyan**
Via: https://github.com/pingcap/docs/pull/1170
@CaitinChen  PTAL

### CODE REVIEW

File: dev/reference/tools/syncer.md [hunk 1]
```diff
@@ -393,53 +393,25 @@ target-table = "order_2017"
...
-            Query OK, 0 rows affected (0.01 sec)
-            ```
-        
-        - 如果 MySQL 有连接，建议重启 MySQL 服务或者杀掉所有连接。
-
-    3. 检查 MySQL `binlog_row_image` 是否为 `FULL`。
+        如果结果是 `log_bin` = `OFF`，则需要开启 binlog，开启方式请参考[官方文档](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html)。
 
-        可以用如下命令检查 `binlog_row_image`：
+    2. binlog 格式必须为 `ROW`，且参数 `binlog_row_image` 必须设置为 `FULL`, 可使用如下命令设置参数：
```
[the remark this answers is not in the document]
Comment (huangxiuyan) [reply dev/reference/tools/syncer.md]: Learned it, thanks!

File: dev/reference/tools/syncer.md [hunk 2]
```diff
@@ -393,53 +393,25 @@ target-table = "order_2017"
...
         ```
 
-        如果 `binlog_row_image` 结果不为 `FULL`，请设置为 `FULL`。设置方式如下：
-
-        ```sql
-        mysql> set global binlog_row_image = FULL;
-        Query OK, 0 rows affected (0.01 sec)
-        ```
+        - 如果以上设置出现错误，可在磁盘上修改配置文件，之后重启 MySQL。
+        - 将配置改变保存在磁盘中很重要，这样 MySQL 重启之后可以显示配置改变。
```
[the remark this answers is not in the document]
Comment (huangxiuyan) [reply dev/reference/tools/syncer.md]: Learned it, thanks!

**Comment by huangxiuyan** [comment]
@yikeke Addressed the comments, PTAL, thanks!

**Comment by huangxiuyan** [comment]
@GregoryIan , PTAL, thanks!

**Comment by GregoryIan** [comment]
LGTM

**Comment by huangxiuyan** [comment]
@lilin90 Could you please help merge this PR? Thanks!

File: dev/reference/tools/syncer.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ binlog-gtid = "2bfabd22-fff7-11e6-97f7-f02fa73bcb01:1-23,61ccbb5d-c82d-11e6-ac2e
 > **注意：**
 >
 > - `syncer.meta` 只需要第一次使用的时候配置，后续 Syncer 同步新的 binlog 之后会自动将其更新到最新的 position。
-> - 如果使用 binlog position 同步则只需要配置 binlog-name binlog-pos; 使用 gtid 同步则需要设置 gtid，且启动 Syncer 时带有 `--enable-gtid`。
+> - 如果使用 binlog position 同步则只需要配置 `binlog-name` 和 `binlog-pos`; 如果使用 `binlog-gtid` 同步则需要设置 `binlog-gtid`，且启动 Syncer 时带有 `--enable-gtid`。
```
Comment (lilin90) [inline dev/reference/tools/syncer.md]:
```suggestion
> - 如果使用 binlog position 同步则只需要配置 `binlog-name` 和 `binlog-pos`；如果使用 `binlog-gtid` 同步则需要设置 `binlog-gtid`，且启动 Syncer 时带有 `--enable-gtid`。
```

File: dev/reference/tools/syncer.md [hunk 4]
```diff
@@ -393,53 +393,25 @@ target-table = "order_2017"
...
-            Query OK, 0 rows affected (0.01 sec)
-            ```
-        
-        - 如果 MySQL 有连接，建议重启 MySQL 服务或者杀掉所有连接。
-
-    3. 检查 MySQL `binlog_row_image` 是否为 `FULL`。
+        如果结果是 `log_bin` = `OFF`，则需要开启 binlog，开启方式请参考[官方文档](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html)。
 
-        可以用如下命令检查 `binlog_row_image`：
+    2. binlog 格式必须为 `ROW`，且参数 `binlog_row_image` 必须设置为 `FULL`, 可使用如下命令查看参数设置：
```
Comment (lilin90) [inline dev/reference/tools/syncer.md]:
```suggestion
    2. binlog 格式必须为 `ROW`，且参数 `binlog_row_image` 必须设置为 `FULL`，可使用如下命令查看参数设置：
```

**Pull request merged**